airflow/cli/commands/celery_command.py

```diff
@@ -18,13 +18,16 @@
 """Celery command."""
 from __future__ import annotations
 
+import logging
 from contextlib import contextmanager
 from multiprocessing import Process
+from sys import stderr as system_stderr, stdout as system_stdout
```

Comment:
Why do these need to be aliased? Doesn’t `import sys` and `sys.stdout` `sys.stderr` do the job?

Reply:
I intended it to be easier distinguishable from `args.stderr` etc.

Inside the `worker` function there is a local variable, which is called `stderr` and when I was working on that I wanted to have them easy to differentiate. 

If it is ok, I will use version with `sys.stderr` instead.

Reply:
Updated.